### PR TITLE
MultiWhisper

### PR DESCRIPTION
**Pull request by funkydemir66**
MultiWhisper 1.0

### CODE REVIEW

**Review by Alynva — CHANGES_REQUESTED** [review]
The icon mush be less than 80x80

File: store/extensions/MultiWhisper/extension.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "title": "Mod Tools",
```
Comment (Alynva) [inline store/extensions/MultiWhisper/extension.json]: The extension title must match the folder name

**Review by Alynva — CHANGES_REQUESTED** [review]

**Comment by funkydemir66** [comment]
okey is finish

**Comment by funkydemir66** [comment]
yh?

**Pull request closed without merging**